website/registrations/admin.py

```diff
@@ -113,9 +120,19 @@ class RegistrationAdmin(admin.ModelAdmin):
             {"fields": ("student_number", "programme", "starting_year",)},
         ),
     )
+    readonly_fields = ("_photo",)
```

Comment:
I think it is better to not mix two ways of setting the read-only fields. This simply be defined inside `get_readonly_fields`